serverless/serverless
# Wrap addPlugin in try/catch statement when loading plugins

## What did you implement:

Closes https://github.com/serverless/serverless/issues/3956

Requiring plugins is now wrapped inside try/catch statement.

## How can we verify it:

serverless.yml
```
plugins:
  - garbage
```

and then run:

```
serverless
```

Following message will be printed:
```
Serverless: WARNING: Serverless plugin "garbage" not found.

Make sure it's installed and spelled correctly in plugins section inside 'serverless.yml' file.
```

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
